packages/client/src/comp/Initialize.tsx

```diff
@@ -30,6 +37,45 @@ export const Initialize: React.FC = () => {
     })()
   }, [oid, setInitialData])
 
+  React.useEffect(() => {
```

Comment:
Can we move this to a new function in `packages/client/src/lib/languages.ts` so that our code is jsut `React.useEffect(() => useLanguageModal(), [])`?

`Initialize` has lots of code from many different places so its code should really be minimal if there's another file we can put this into.